app/assets/stylesheets/conversation_show.scss

```diff
@@ -67,12 +106,19 @@ margin: 0 0 5px 0;
   // margin: 0 0 2px 10px;
 }
 
-.message-show-container {
+.chatboxcontent {
   height:95%;
   overflow:auto;
   background:#fff;
+  line-height: 1.3em;
+  list-style: none;
 }
 
+.chatboxcontent div {
+    padding: 0.5rem;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered display, overflow, padding